### DISCUSSION TITLE
Deployments stuck in queued

### DISCUSSION THREAD

**Discussion by mikeborozdin** [vercel/community#4251] [Help]
### Summary

Hi,

For the last hour the deployments have been stuck  `Queued` status. There's always a single deployment at the time.

I cannot deploy anything to production.

The status page does not show ay issues.

I'm on the Pro account

### Steps to Reproduce

- Push to GitHub
- Look at the deployment status
- It'll be stuck in `queued.'

**Comment by ofhouse** [comment]
We had an issue with builds for Hobby & Pro customers and are actively working on mitigation.
Please see more information on our status page: https://www.vercel-status.com/incidents/87m1ypq44hf0
